File: plus/8_draw_geometry2d.py
import harfang as hg

hg.LoadPlugins()

plus = hg.GetPlus()
plus.RenderInit(400, 300)

angle = 0
cube = plus.CreateGeometry(plus.CreateCube())

while not plus.IsAppEnded():
	plus.Clear()
	plus.Geometry2D(200, 150, cube, angle, angle * 2, 0, 100)
	plus.Flip()
	plus.EndFrame()

	angle += 0.01

plus.RenderUninit()

File: plus/8_draw_geometry2d.lua
plus = gs.GetPlus()
plus:RenderInit(400, 300)

angle = 0
cube = plus:CreateGeometry(plus:CreateCube())

while not plus:KeyPress(gs.InputDevice.KeyEscape) do
	plus:Clear()
	plus:Geometry2D(200, 150, cube, angle, angle * 2, 0, 100)
	plus:Flip()

	angle = angle + 0.01
end
